sources/core/objects/Grille.cpp

```diff
@@ -37,15 +37,36 @@ void Grille::AttackBy(IdPtr<Item> item)
             SetPassable(D_ALL, Passable::FULL);
             cutted_ = true;
             Create<IOnMapObject>(Rod::T_ITEM_S(), GetOwner());
+            return;
         }
         else
         {
             Create<IOnMapObject>(Rod::T_ITEM_S(), GetOwner());
             Delete();
+            return;
         }
     }
+    Breakable::AttackBy(item);
```

Comment:
It is too early for `AttackByHand` function (because there is possibility that some special item named `Hand` will be added, and each `Human` will own such `Hand` and so on).
What changes are needed for `AttackBy` in `Human`?